package.json

```diff
@@ -121,6 +121,7 @@
     "@visx/xychart": "^2.12.2",
     "@wagmi/core": "^0.5.5",
     "@walletconnect/web3-provider": "^1.7.8",
+    "@wherever/react-notification-feed": "^0.0.11",
```

Comment:
@nivo33 can we please see the source code for this before we integrate it? i can see it on npm but no link to github https://www.npmjs.com/package/@wherever/react-notification-feed?activeTab=versions

Reply:
i can see this contains analytics with hardcoded keys